### ISSUE TITLE
Loading placeholder keep flashing and webbrowser keep moving because of the loading bar

### ISSUE THREAD

**Issue by cylian914** [oceancabbage/RimSort#70]
The place holder for loading the website keep flashing, can it be just remove or only load when the webbrowser start ?
https://user-images.githubusercontent.com/73445993/236800763-77540eb2-a20d-4bff-9296-f5c972d1cf40.mp4

Also the loading bar keep disappearing, the bar should always  be there even if it's invisible

**Comment by twstagg** [comment]
Per discussion, i'm open to replacing this image. We'll keep it as-is for now. I personally think it looks a bit better than the white background the webview has during that stage of loading the page.

**Comment by hk21702** [comment]
It might be better to just do a solid shade that is similar to that of the background of steam? Then you get less sudden bright lights.

**Comment by EttyKitty** [comment]
> Also the loading bar keep disappearing, the bar should always be there even if it's invisible

I can offer reduced, fixed 8px height; the bar is always displayed; the % is only displayed when loading; the progress color is changed to fit steam colors; the font size is decreased.
The text doesn't look very good, but how much do you care. Can remove it completely.

<details>
<summary>Screenshots</summary>

<img width="1239" height="85" alt="Image" src="https://github.com/user-attachments/assets/02a8c7cc-0612-413d-b843-6540200264c0" />

<img width="1707" height="65" alt="Image" src="https://github.com/user-attachments/assets/eaabd2fa-1383-47c0-b848-6db93ab7a7d5" />

</details>

> The place holder for loading the website keep flashing, can it be just remove or only load when the webbrowser start ?

Here, just a solid color looks okay imo.

<details>
<summary>Screenshot</summary>

<img width="1709" height="780" alt="Image" src="https://github.com/user-attachments/assets/cd042460-d273-4232-a91b-9c7ff4473bbe" />

</details>

You can simply remove `setPixmap` and replace `ImageLabel` with `QLabel`, and it'll just use the current theme color. I think there are no caveats?

**Comment by EttyKitty** [comment]
Can this issue count as solved now that #1020 was merged?

https://github.com/user-attachments/assets/c64b46e6-fdf4-4812-8ca6-203e11535f72

**Comment by LionelColaso** [comment]
> Can this issue count as solved now that [#1020](https://github.com/RimSort/RimSort/pull/1020) was merged?
> 2025-07-15.08-49-33.mp4

Yes Thankyou I thought the issue was tagged and closed along with merge
Closing Now
